server/modules/organisation/use-cases/delete-organisation.js

```diff
@@ -0,0 +1,7 @@
+import * as Organisation from '../model';
+
+function deleteOrganisation({ id }) {
+  return Organisation.updateOrganisationToDeleted(id);
+}
+
+export default deleteOrganisation;
```

Comment:
doesn't match the same file in CoL
Make sure to go through all the backend files again please